### DISCUSSION TITLE
How can I add keyboard to my bot?

### DISCUSSION THREAD

**Discussion by mal1k** [skrtdev/NovaGram#4] [Q&A] [answered]
How can I use ReplyKeyboardMarkup And InlineKeyboard ?
May u send some examples please?

**Comment by skrtdev** [comment, marked as answer]
Check https://github.com/skrtdev/NovaGram/blob/master/examples/encoder_bot.php

**Comment by Jobians** [comment]
$user->sendMessage("Hiiiii", [
        'reply_markup' => ["keyboard" => [[["text" => "🎮 Games"]],
            [["text" => "🙌🏻 Referral"],
                ["text" => "💰 Balance"]],
            [["text" => "📥 Withdraw"],
                ["text" => "💵 Deposit"]],
            [["text" => "📜 Bet Rule"],
                ["text" => "⚖️ Fairness"]]], "resize_keyboard" => true]
    ], true);
